README.md

```diff
@@ -1,318 +1,194 @@
-
-### A simple example
-
-```ocaml
-let port = 3000
-let hostname = "127.0.0.1"
-let create_server http  = 
-  let server = http##createServer begin fun [@bs] req  resp  -> 
-      resp##statusCode #= 200;
-      resp##setHeader "Content-Type" "text/plain";
-      resp##_end "Hello world\n"
-    end
-  in
-  server##listen port hostname  begin fun [@bs] () -> 
-    Js.log ("Server running at http://"^ hostname ^ ":" ^ Pervasives.string_of_int port ^ "/")
-  end 
-
-
-let () = 
-  create_server Http_types.http
-```
-Generated by BuckleScript
-```js
-'use strict';
-var Pervasives = require("bs-platform/lib/js/pervasives");
-var Http       = require("http");
-
-var hostname = "127.0.0.1";
-
-function create_server(http) {
-  var server = http.createServer(function (_, resp) {
-        resp.statusCode = 200;
-        resp.setHeader("Content-Type", "text/plain");
-        return resp.end("Hello world\n");
-      });
-  return server.listen(3000, hostname, function () {
-              console.log("Server running at http://" + (hostname + (":" + (Pervasives.string_of_int(3000) + "/"))));
-              return /* () */0;
-            });
-}
-
-create_server(Http);
-```
-
-You can download the example [here](https://github.com/bloomberg/bucklescript-addons/tree/master/examples/node-http-server):
-
-
-As you can see, there is no name mangling in the generated code, the function `string_of_int` is from a normal OCaml module `Pervasives`, `Pervasives.string_of_int` is directly callable from other JavaScript code.
+## Try BuckleScript
 
-The attribute `[@bs]` allows BuckleScript to support JS's uncurried calling convention in addition to OCaml's curried calling convention.
+You can try BuckleScript directly [in your browser](http://bloomberg.github.io/bucklescript/js-demo/). Write OCaml in the left panel and
+watch as it instantly compiles to JavaScript shown in the right panel.
 
+## Installing BuckleScript
 
+#### Pre-Requisites
 
-## Disclaimer
+* Standard C/C++ toolchain
+* Node 4 or newer
 
-It is in an *beta* stage, but we encourage you to try it and share your feedback.
-
-## Documentation
-
-See http://bloomberg.github.io/bucklescript. If you want to contribute documentation, the source is [here](./docs).
-
-## More examples
-
-See https://github.com/bloomberg/bucklescript-addons
-
-## Build
-
-
-### Linux and Mac OSX
-
-#### Build from package manager
-
-#. npm install from npmjs registry
+#### Installing the Release Package
 
 ```
-npm install  bs-platform
+npm install bs-platform
 ```
 
-It will install OCaml compiler, BuckleScript compiler and Standard library
-in three module systems (CommonJS, AMDJS, and Google Module).
-
-#. npm install from github (latest one)
+#### Installing from Source
 
 ```
-npm install git+https://github.com/bloomberg/bucklescript/
+git clone https://github.com/bloomberg/bucklescript
+cd bucklescript
+npm install
 ```
 
-#### Build from source
```

Comment:
It would be nice to still keep a section about building from source, so that people can hacking the compiler.